@here/harp-mapview/lib/Tile.ts

```diff
@@ -347,6 +347,13 @@ export class Tile implements CachedResource {
      */
     levelOffset: number = 0;
 
+    /**
+     * @hidden
+     *
+     * Background mesh used for the fallback logic.
+     */
+    backgroundPlane?: THREE.Mesh;
```

Comment:
does this need to be public?

Reply:
i changed it to be a get / set accessor, because I had the addBackgroundPlane and then this public, so now it should be a bit cleaner interface.

It is needed in the VisibleTileSet and the TileGeometryCreator